server/files.go

```diff
@@ -366,7 +368,7 @@ func getFileContentsEndpoint(s Service, logger log.Logger) endpoint.Endpoint {
 			return getFileContentsResponse{Err: ErrFoundABug}, ErrFoundABug
 		}
 
-		r, err := s.GetFileContents(req.ID)
+		r, err := s.GetFileContents(req.ID, req.opts)
```

Comment:
Don't we need to read LineEnding from the request? From a query param or Accept header?

Reply:
@adamdecaf yes, we would need to take it from input. Either a param or a header could work.

I was under the impression this option was being parsed automatically from the body, but I could very well be wrong. I can add a test to try and confirm.

Reply:
@adamdecaf thanks, I revisited this and understand your point now.

I added an HTTP header to convey this information and added tests verifying the behavior.

_Related to this -- what is the process for propagating changes from the openapi spec here to the generated client libraries?_

Reply:
I'm wondering if we should accept "CRLF" as an preference over the header accepting `\r` / `\n` as they may get wiped out by HTTP servers.

Reply:
@adamdecaf I am inclined to agree.

I think an enum-like value accepting `CRLF | LF` would be a good choice.